### DISCUSSION TITLE
请问：注入结果：0 ，是什么原因

### DISCUSSION THREAD

**Discussion by yanweidong** [ttttupup/wxhelper#128] [General]
C:\ wechat\WeChat\[3.9.2.23]>ConsoleApplication.exe -i WeChat.exe -p wxhelper.dll
注入结果:0

**Comment by ttttupup** [comment]
注入失败了，这个要看下lasterror。

**Reply by yanweidong** [reply]
大神，有微信群或是别的群嘛。github太不好上传图片，我想和大家成功的沟通一下。感谢

**Reply by ttttupup** [reply]
没有

**Reply by ttttupup** [reply]
https://t.me/+LmvAauweyUpjYzJl

**Reply by Don5211314** [reply]
> 大神，有微信群或是别的群嘛。github太不好上传图片，我想和大家成功的沟通一下。感谢

你好 你有联系方式吗 有些问题想请教

**Reply by qianqian1530** [reply]
vx : HaiTang_PP      小白建个群 一起研究啊

**Reply by furryaxw** [reply]
> vx : HaiTang_PP 小白建个群 一起研究啊

微信里讨论微信机器人是嫌死的不够快吗
